### DISCUSSION TITLE
Add a Matrix Space for `inquire`

### DISCUSSION THREAD

**Discussion by kajuburfi** [mikaelmello/inquire#58] [Ideas]
**Is your feature request related to a problem? Please describe.**
Just a better place for chats, and simple questions that don't require separate issues.

**Describe the solution you'd like**
How about having a [Matrix](https://matrix.org/) space. (I use [Element](https://element.io/))

**Comment by tpoliaw** [comment]
There is also the discussions feature GitHub offers if there's a benefit to keeping questions with the code

**Comment by mikaelmello** [comment]
I think having a proper forum for discussions ins a great idea. There are a tons of good options e.g. Matrix, Zulip, Discourse, etc., but I like @tpoliaw. I don't think this project is big enough to warrant a forum on another website.

As a matter of fact, I once had discussions opened but closed because no one was here back then 😃

**Comment by mikaelmello** [comment]
I'll configure the Discussions tab in this repo

**Comment by mikaelmello** [comment]
Done, thanks for the suggestions!

I'll leave this open as an idea for a Matrix space, if it gains popularity I don't see why not.
